### ISSUE TITLE
[FEATURE REQUEST] Portable Mode

### ISSUE THREAD

**Issue by SoloBackup** [jpd002/Play-#1177]
Hello Goodnight.
Much appreciated with the work and the gains that have been made recentely.
I would suggest a portable mode. Where the "Play Data Files" folder would be in the same directory as the Play! executable, as in other projects.
I'm using it on windows.
Thank you for what you have done in the community and I wish you success!

**Comment by Tartifless** [comment]
Hello, i support this request, it would be very nice-to-have.
here is a way some other emulators do this:

- Add this in the emulator options
- Enable this based on a "portable.ini" file located in the emulator folder
- Enable this based on the presence of the "Play Data Files" folder in the emulator folder
- use a system variable
- Enable this by using a command line argument like "-localdata"

Regards

**Comment by Nicktamer** [comment]
Good idea, more practical to use. All other emulator have a portable option. For using in front end or for use in ssd on other computers just have to copy one folder. Locking "play data files" in documents is not "practical "

**Comment by Aynshe** [comment]
Option that would be appreciated :)

**Comment by SpiralBrad** [comment]
I just discovered the **RetroArch Play! core** writes files to %USERPROFILE%\Documents\Play Data Files\ on Windows.

It should definitely stick to RetroArch (LibRetro(?)) convention when used in that context, using the saves and systems folders for such things.
